cg/cli/workflow/nallo/base.py

```diff
@@ -27,3 +31,20 @@ def nallo(context: click.Context) -> None:
 nallo.add_command(run)
 nallo.add_command(start)
 nallo.add_command(metrics_deliver)
+
+
+@nallo.command("panel")
+@DRY_RUN
+@ARGUMENT_CASE_ID
+@click.pass_obj
+def panel(context: CGConfig, case_id: str, dry_run: bool) -> None:
+    """Write aggregated gene panel file exported from Scout."""
+
+    analysis_api: NalloAnalysisAPI = context.meta_apis["analysis_api"]
+    analysis_api.status_db.verify_case_exists(case_internal_id=case_id)
+
+    bed_lines: list[str] = analysis_api.get_gene_panel(case_id=case_id)
+    if dry_run:
+        echo_lines(lines=bed_lines)
```

Comment:
here the lines starting with "##" will not be excluded. If this is intended feel free to ignore

Reply:
Yes correct, here it is not removed yet.